src/main/webapp/onelive/archive/functions.js

```diff
@@ -36,3 +36,9 @@ function loadYoutubeVideoPlayer(videoId) {
     let iframeTag = '<iframe class="embed-responsive-item" src="https://www.youtube.com/embed/' + videoId + '"></iframe>';
     $('#iframe').html(iframeTag);
 }
+
+function loadHomePageVideo() {
+    let videoId = window.location.href.substr(1).split('videoId=')[1];
```

Comment:
1st Check if there's a param exists. (there's no videoId param if someone just visit the archive page)

use this way: https://stackoverflow.com/questions/7731778/get-query-string-parameters-url-values-with-jquery-javascript-querystring

Reply:
I'll fix it.